### PR TITLE
[EDU-4150] update nextjs dedicated page

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue:  https://aziontech.atlassian.net/browse/EDU-4150

Changes: 

- Add table of compatibility
- Restructure docs
- Create new permalink since the original had a redirect

### CODE REVIEW

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 1]
```diff
@@ -1,9 +1,79 @@
 ---
-title: Next.js
-description: Next.js is a flexible React framework that helps you to build fast and modern applications.
-meta_tags: nextjs, jamstack, javascript
-namespace: documentation_products_edge_functions_runtime_supported_frameworks_nextjs
-permalink: /documentation/devtools/runtime-api/supported-frameworks/nextjs/
+title:  Azion and Next.js compatibility
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:2]:
```suggestion
title: Azion and Next.js compatibility
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 2]
```diff
@@ -1,9 +1,79 @@
...
+description: Explore the compatibility of different versions of Next.js with Azion Edge Platform.
+meta_tags: Next.js, Azion Edge Platform, Compatibility, Static Pages, SSR, SSG, Edge API Routes, Dynamic Routes, Middleware, Next configs, i18n routing, Custom errors, React Framework, Modern Applications, Routing, Data Fetching, Integrations
+namespace: documentation_devtools_runtime_compatibility_nextjs
+permalink: /documentation/devtools/azion-edge-runtime/compatibility/nextjs/
+---
+
+Compatibility between different versions of Next.js and Azion Edge Platform.
+
+:::note
+ You can find more detailed examples in the [Vulcan repository](https://github.com/aziontech/vulcan/tree/main/examples/next).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:12]:
```suggestion
You can find more detailed examples in the [Vulcan repository](https://github.com/aziontech/vulcan/tree/main/examples/next).
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 3]
```diff
@@ -1,9 +1,79 @@
...
+### Versions: 12.2.x, 12.3.x
+
+| Pages Router Support | Support | Comments |
+|---|---|---|
+| Static Pages | 🟢 | |
+| SSR | 🟢 | |
+| SSG | 🟢 | |
+| Edge API Routes | 🟢 | |
+| Dynamic Routes | 🟢 | |
+| Middleware  | 🟡  | Features: rewrite, redirect, continue to response, set request header, throw error, set response header and set response cookie|
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:28]:
```suggestion
| Middleware  | 🟡  | Features: rewrite, redirect, continue to response, set request header, throw error, set response header, and set response cookie|
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 4]
```diff
@@ -1,9 +1,79 @@
...
+| Next configs | 🟡  | Features: rewrite before files, rewrite after files, rewrite fallback, redirects and header definition  |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:29]:
```suggestion
| Next configs | 🟡 | Features: rewrite before files, rewrite after files, rewrite fallback, redirects, and header definition  |
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 5]
```diff
@@ -1,9 +1,79 @@
...
+| i18n routing | 🟢 | |
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:32]:
```suggestion
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 6]
```diff
@@ -1,9 +1,79 @@
...
+### Versions: 13.0.x, 13.1.x, 13.2.x, 13.3.x, 13.4.x, 13.5.x
+
+| Pages Router Support | Support | Comments |
+|---|---|---|
+| Static Pages | 🟢 | 
+| SSR | 🟢 | | 
+| SSG | 🟢 | | 
+| Edge API Routes | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Middleware | 🟡  | Features: rewrite, redirect, continue to response, set request header, throw error, return response, set response header and set response cookie |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:42]:
```suggestion
| Middleware | 🟡 | Features: rewrite, redirect, continue to response, set request header, throw error, return response, set response header, and set response cookie | 
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 7]
```diff
@@ -1,9 +1,79 @@
...
+
+| Pages Router Support | Support | Comments |
+|---|---|---|
+| Static Pages | 🟢 | 
+| SSR | 🟢 | | 
+| SSG | 🟢 | | 
+| Edge API Routes | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Middleware | 🟡  | Features: rewrite, redirect, continue to response, set request header, throw error, return response, set response header and set response cookie | 
+| Next configs | 🟡  | Features: rewrite before files, rewrite after files, rewrite fallback, redirects, header definition |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:43]:
```suggestion
| Next configs | 🟡 | Features: rewrite before files, rewrite after files, rewrite fallback, redirects, and header definition | 
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 8]
```diff
@@ -1,9 +1,79 @@
...
+| Middleware | 🟡  | Features: rewrite, redirect, continue to response, set request header, throw error, return response, set response header and set response cookie | 
+| Next configs | 🟡  | Features: rewrite before files, rewrite after files, rewrite fallback, redirects, header definition | 
+| i18n routing | 🟢 | | 
+| Custom errors | 🟢 | | 
+
+<br />
+
+| App Router Support | Support | Comments |
+|---|---|--- | 
+| App router | 🟢 | Basic structure, routing and layouts|
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:51]:
```suggestion
| App router | 🟢 | Basic structure, routing, and layouts| 
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 9]
```diff
@@ -1,9 +1,79 @@
...
+
+<br />
+
+| App Router Support | Support | Comments |
+|---|---|--- | 
+| App router | 🟢 | Basic structure, routing and layouts| 
+| Server Components | 🟢 | | 
+| Route Handlers | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Middleware  | 🟡  | Features: rewrite, redirect, continue to response, set request header, throw error, set response header and set response cookie|
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:55]:
```suggestion
| Middleware  | 🟡 | Features: rewrite, redirect, continue to response, set request header, throw error, set response header, and set response cookie|
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 10]
```diff
@@ -1,9 +1,79 @@
...
+| Next configs | 🟡  | Features: rewrite before files, rewrite after files, rewrite fallback, redirects and header definition  |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:56]:
```suggestion
| Next configs | 🟡 | Features: rewrite before files, rewrite after files, rewrite fallback, redirects, and header definition  |
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx [hunk 11]
```diff
@@ -1,9 +1,79 @@
...
+### Versions: 12.3.x
+
+| Pages Router Support | Support | Comments |
+|---|---|---| 
+| Static Pages | 🟢 | | 
+| SSR | 🟢 | | 
+| SSG | 🟢 | | 
+| API Routes | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Next configs  | 🟡  | Features: rewrite before files, rewrite after files, rewrite fallback, redirects and header definition |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/nextjs/2023-02-14-index.mdx:73]:
```suggestion
| Next configs | 🟡 | Features: rewrite before files, rewrite after files, rewrite fallback, redirects, and header definition | 
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 12]
```diff
@@ -1,12 +1,81 @@
 ---
-
-title: Next.js
-description: Next.js é um framework React que auxilia no desenvolvimento de aplicacões rápidas e modernas.
+title: Compatiblidade entre a Azion e Next.js
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:2]:
```suggestion
title: Compatiblidade entre a Azion e o Next.js
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:2]: fiquei na dúvida se deve levar

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 13]
```diff
@@ -1,12 +1,81 @@
...
 meta_tags: nextjs, jamstack, javascript
-namespace: documentation_products_edge_functions_runtime_supported_frameworks_nextjs
+namespace: documentation_devtools_runtime_compatibility_nextjs
 permalink: /documentacao/devtools/runtime-api/frameworks-suportados/nextjs/
 ---
 
+Compatibilidade entre diferentes versões do Next.js e a Plataforma Edge da Azion.
+
+:::note
+ Você pode encontrar exemplos mais detalhados no [repositório Vulcan](https://github.com/aziontech/vulcan/tree/main/examples/next).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:12]:
```suggestion
 Você pode encontrar exemplos mais detalhados no [repositório Vulcan](https://github.com/aziontech/vulcan/tree/main/examples/next).
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 14]
```diff
@@ -1,12 +1,81 @@
...
+### Versões: 12.2.x, 12.3.x
+
+| Suporte Pages Router | Suporte | Comentários |
+|---|---|---|
+| Static Pages | 🟢 | |
+| SSR | 🟢 | |
+| SSG | 🟢 | |
+| Edge API Routes | 🟢 | |
+| Dynamic Routes | 🟢 | |
+| Middleware  | 🟡  | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, set response header e set response cookie|
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:28]:
```suggestion
| Middleware | 🟡 | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, set response header e set response cookie |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 15]
```diff
@@ -1,12 +1,81 @@
...
+| Next configs | 🟡  | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition  |
+| i18n routing | 🟢 | |
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:31]:
```suggestion
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 16]
```diff
@@ -1,12 +1,81 @@
...
+| Suporte Pages Router | Suporte | Comentários |
+|---|---|---|
+| Static Pages | 🟢 | |
+| SSR | 🟢 | |
+| SSG | 🟢 | |
+| Edge API Routes | 🟢 | |
+| Dynamic Routes | 🟢 | |
+| Middleware  | 🟡  | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, set response header e set response cookie|
+| Next configs | 🟡  | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition  |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:29]:
```suggestion
| Next configs | 🟡 | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 17]
```diff
@@ -1,12 +1,81 @@
...
+### Versões: 13.0.x, 13.1.x, 13.2.x, 13.3.x, 13.4.x, 13.5.x
+
+| Suporte Pages Router | Suporte | Comentários |
+|---|---|---|
+| Static Pages | 🟢 | 
+| SSR | 🟢 | | 
+| SSG | 🟢 | | 
+| Edge API Routes | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Middleware | 🟡  | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, return response, set response header e set response cookie |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:42]:
```suggestion
| Middleware | 🟡 | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, return response, set response header e set response cookie | 
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 18]
```diff
@@ -1,12 +1,81 @@
...
+
+| Suporte Pages Router | Suporte | Comentários |
+|---|---|---|
+| Static Pages | 🟢 | 
+| SSR | 🟢 | | 
+| SSG | 🟢 | | 
+| Edge API Routes | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Middleware | 🟡  | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, return response, set response header e set response cookie | 
+| Next configs | 🟡  | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects, header definition |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:43]:
```suggestion
| Next configs | 🟡 | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition | 
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 19]
```diff
@@ -1,12 +1,81 @@
...
+<br />
+
+| Suporte App Router | Suporte | Comentários |
+|---|---|--- | 
+| App router | 🟢 | Basic structure, routing e layouts| 
+| Server Components | 🟢 | | 
+| Route Handlers | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Middleware  | 🟡  | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error, set response header e set response cookie|
+| Next configs | 🟡  | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition  |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:56]:
```suggestion
| Middleware  | 🟡 | Funcionalidades: rewrite, redirect, continue to response, set request header, throw error e set response header e set response cookie |
| Next configs | 🟡 | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition  |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx [hunk 20]
```diff
@@ -1,12 +1,81 @@
...
+### Versões: 12.3.x
+
+| Suporte Pages Router | Suporte | Comentários |
+|---|---|---| 
+| Static Pages | 🟢 | | 
+| SSR | 🟢 | | 
+| SSG | 🟢 | | 
+| API Routes | 🟢 | | 
+| Dynamic Routes | 🟢 | |
+| Next configs  | 🟡  | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/nextjs/nextjs.mdx:73]:
```suggestion
| Next configs  | 🟡 | Funcionalidades: rewrite before files, rewrite after files, rewrite fallback, redirects e header definition | 
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**